### PR TITLE
FFTF-267: Farming type error text update

### PR DESCRIPTION
**Pull request by diya2214**

### CODE REVIEW

File: app/routes/farming-type.js [hunk 1]
```diff
@@ -14,24 +16,7 @@ function createModel (errorMessage, data) {
...
-          text: 'Horticulture (including ornamentals)',
-          checked: !!data && (data.includes('Horticulture'))
-        },
-        {
-          value: 'Something else',
-          text: 'Something else',
-          checked: !!data && (data.includes('Something else'))
-        }
-      ],
+      items: setLabelData(data, ['Crops for the food industry', 'Horticulture (including ornamentals)', 'Something else']),
```
Comment (ogbk) [inline app/routes/farming-type.js:19]: this means that value of second  radiobutton is no longer 'Horticulture', but now 'Horticulture (including ornamentals)' - I think the value should be simple and remain 'Horticulture'

File: app/routes/farming-type.js [hunk 2]
```diff
@@ -63,7 +48,12 @@ module.exports = [
         payload: Joi.object({
           farmingType: Joi.string().required()
         }),
-        failAction: (request, h) => h.view('farming-type', createModel('Please select an option')).takeover()
+        failAction: (request, h, err) => {
+          const errorObject = errorExtractor(err)
+          console.log(errorObject)
```
Comment (ogbk) [inline app/routes/farming-type.js:53]: we shouldn't have logs in the code

File: app/routes/farming-type.js (hunk 1 above)
Comment (diya2214) [reply app/routes/farming-type.js:19]: @ogbk helper function updated to resolve your comments .

File: app/helpers/helper-functions.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ function isChecked (data, option) {
 function setLabelData (data, labelData) {
   return labelData.map((label) => {
     return {
-      value: label,
+      value: label.split('(')[0],
```
Comment (ogbk) [inline app/helpers/helper-functions.js:10]: this can easily lead into errors when setting values

**Review by ogbk — APPROVED** [review]

**Pull request merged**